### ISSUE TITLE
mail failed to send with message:  NumberFormatException: For input string: "Request"

### ISSUE THREAD

**Issue by hbwilson1** [k9mail/k-9#3332]
For no apparent reason, mail sent from one of three accounts is giving the error message: NumberFormatException: for input string: "Request"

Should be sending normally. Password and user ID have not changed. These are working fine on other email clients.

Send emails, exception appears. All other accounts functioning well. 

K-9 Mail version: 5.403

Android version: 7.0. Patched March 1, 2018

Account type (IMAP, POP3, WebDAV/Exchange): POP3

**Comment by philipwhiuk** [comment]
This is really odd. Logs would be very useful here.

**Comment by hbwilson1** [comment]
Logging is enabled. How do i locate and upload file?

**Comment by cketti** [comment]
See https://github.com/k9mail/k-9/wiki/LoggingErrors

**Comment by hbwilson1** [comment]
Trying to follow the LoggingErrors page... item 2 says "find process..." using this command sting"
   adb shell ps -A | grep k9
Message comes back
   "'grep' is not recognized as an internal or external command, operable program or batch file.

Is there a typo in instructions? Has adb command been changed since this was written?

**Comment by philipwhiuk** [comment]
`grep` is a *nix command available on Linux and Mac terminal. 

I believe `findstr` performs similarly on Windows.

i.e. 

`adb shell ps -A | findstr k9` 

should work. I'll update the wiki.

**Comment by hbwilson1** [comment]
Tried
    adb shell ps -A | findstr k9

adb did something (hesitated), but returned to command prompt without giving any value.

**Comment by hbwilson1** [comment]
verified | (pipe in a cmd prompt window as shown in above screen shot.

**Comment by hbwilson1** [comment]
verified | (pipe in a cmd prompt window as shown in above screen shot.

**Comment by cketti** [comment]
No information provided.

**Comment by jagman653** [comment]
Just want to report that I recently got this error with the F-Droid K-9. It was due to a recent AT&T change to requiring Secure Mail Key. I used the procedure prescribed by AT&T to create an SMT and changed the password for my SBCGlobal K-9 account to that SMT. Afterwards, K-9 automatically set up the account and I was able to send & receive mail!

BTW, the K-9 automatic procedure set the server names to the AT&T server names.
